### PR TITLE
Create weekly-update.2015-06-26.md

### PR DESCRIPTION
**Pull request by Gioyik**
CC #101

### CODE REVIEW

File: weekly-updates/weekly-update.2015-06-26.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* Slide deck: [Bluemix Webinar: Deploying a Full Stack Node.js Application to IBM Bluemix](https://speakerdeck.com/bradleyholt/bluemix-webinar-deploying-a-full-stack-node-dot-js-application-to-ibm-bluemix)
+* Article by RisingStack: [How to Use Rust with Node.js When Performance Matters](http://blog.risingstack.com/how-to-use-rust-with-node-when-performance-matters/)
+* Device Atlas API now [supports Node.js](https://deviceatlas.com/blog/deviceatlas-api-node-js?utm_source=twitter&utm_medium=update&utm_campaign=node%20js%20support)
+* [On Maintaining a Native Node Module](http://www.voodootikigod.com/on-maintaining-a-native-node-module/) by Chris Williams
+* [nodei.co](http://twitter.com/rvagg/status/613688739030679552) is running with io.js
+* npm [3.0.0 pre-release](https://github.com/npm/npm/releases/tag/v3.0.0)
+
+### Upcoming Events
+
+* [Nodeday](http://nodeday.com) tickets are free, June 26th at DigitasLBi, London
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-06-26.md]: I think this event is finished.

**Comment by c0b41** [comment]
@Gioyik  @yosuke-furukawa  status?

**Comment by Gioyik** [comment]
This is ready to merge

**Comment by yosuke-furukawa** [comment]
yay.

**Comment by yosuke-furukawa** [comment]
merged. I will publish this after my company work.

**Pull request merged**